onnxruntime/core/providers/qnn/builder/qnn_backend_manager.cc

```diff
@@ -2094,5 +2108,72 @@ Status QnnBackendManager::GetOrRegisterContextMemHandle(Qnn_ContextHandle_t cont
   return Status::OK();
 }
 
+Status QnnBackendManager::InvokeWithSSRHandle(
+    const std::function<Status()>& operation,
+    const std::function<Status()>& ssr_recover,
```

Comment:
Can these functions throw exceptions and what effect it would have on the state machine?